### PR TITLE
Base image update @ 24-05-2021

### PR DESCRIPTION
**Pull request by priitlatt**
Base image update @ 24-05-2021

### CODE REVIEW

File: content/specs/versions-linux.md [hunk 1]
```diff
@@ -29,6 +29,9 @@ weight: 5
 - gem `3.1.4`
 - gh `1.8.1`
 - git `2.25.1`
+- Google Cloud SDK `325.0.0`
+- gsutil `4.58`
+
```
Comment (priitlatt) [inline content/specs/versions-linux.md:34]: Excess empty line?
```suggestion
```

File: content/specs/versions.md [hunk 2]
```diff
@@ -32,7 +32,9 @@ weight: 4
 - gem `3.0.3`
 - gh `1.8.1`
 - git `2.28.0`
+- Google Cloud SDK `270.0.0`
```
Comment (craastad) [inline content/specs/versions.md:35]: Why not add the command, `gcloud`? Or are there a set of commands that come with this?

**Comment by craastad** [comment]
Hi. Have these changes been released on Codemagic's side? I'm waiting for this to respond to https://github.com/codemagic-ci-cd/codemagic-docs/pull/364/files

**Comment by helinanever** [comment]
@craastad the docs have been already updated with the tools added in this PR, I feel safe closing this PR.

**Comment by craastad** [comment]
Closing as newer PR https://github.com/codemagic-ci-cd/codemagic-docs/pull/370 was merged

**Pull request closed without merging**